img/images.qrc

```diff
@@ -83,6 +83,7 @@
         <file>actions/flip_horizontal.png</file>
         <file>actions/flip_vertical.png</file>
         <file>app/librepcb.png</file>
+        <file>app/librepcb64.png</file>
```

Comment:
Why a new image, isn't it the same as librepcb.png?

Reply:
It's scaled down to 64x64px. Qt apparently cannot scale background images.

Reply:
Normally I use a QLabel (not QWidget) for displaying images. Just set the "pixmap" and "scaledContents" properties and set the label to a fixed size.

Reply:
Done